webpack.config.js

```diff
@@ -56,7 +56,6 @@ module.exports = {
 	plugins: [
 		new DefinePlugin( {
 			// Inject the `GUTENBERG_PHASE` global, used for feature flagging.
-			// eslint-disable-next-line @wordpress/gutenberg-phase
```

Comment:
@talldan - any ideas why it would be no longer necessary?

Reply:
@gziolo - I must've improved the linting rule at some point, and not removed this. I think removing it is good, the linting rule still seems to catch the intended things.

Reply:
I think that rule only checks usages of `process.env.GUTENBERG_PHASE` in the code, while here it is used in a string.